index.js

```diff
@@ -17,7 +37,7 @@ module.exports = {
       patterns: [
         {
           match: /process\.env\.NODE_ENV/g,
-          replacement: "'production'"
+          replacement: `"${app.env}"`
```

Comment:
@sandersky I love reading all your PRs because I'm learning so much about ember-cli/addons/hooks I didn't even know about :)

If @MiguelMadero says it's g2g I'll second :)